stanford_image.features.field_instance.inc

```diff
@@ -27,94 +27,256 @@ function stanford_image_field_default_field_instances() {
       'stanford_3_col_header' => array(
         'label' => 'hidden',
         'module' => 'text',
-        'settings' => array(),
+        'settings' => array(
```

Comment:
Not sure which contrib module these are from but it may be. That or Display suite with some sort of configuration.